docker_scripts/config-docker-sat.sh

```diff
@@ -76,12 +76,10 @@ import yaml
 
 with open("${METAL_PROVISION_BASE_PACKAGES_PATH}") as pkgs:
     pkgs = yaml.safe_load(pkgs)
-    for line in pkgs.get("packages", []):
-        name, version = re.split(r'(?:(?:<|>)=?|=)', line)
-        if name == "kubectl":
-            version = semver.VersionInfo.parse(version).finalize_version()
-            print(str(version))
-            sys.exit(0)
+    version = pkgs.get("kubernetes_release")
```

Comment:
With this simplification, we no longer need the `import re` on line 70 above and the `import semver` on line 73 above. Please go ahead and remove those.